statsmodels/tsa/adfvalues.py

```diff
@@ -383,5 +383,387 @@ def mackinnoncrit(N=1, regression ="c", nobs=inf):
     else:
```

Comment:
Not part of this PR, but the `ValueError` in 380 needs to have the close-parens moved.  As-is it will raise `TypeError: unsupported operand type(s) for %: 'exceptions.ValueError' and 'str'`